### PR TITLE
Forms Automated Test - LPS-133134 | master

### PR DESCRIPTION
**Pull request by leticiachiu**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsColorField.testcase [hunk 1]
```diff
@@ -76,13 +76,44 @@ definition {
...
+		FormViewBuilder.validateFieldLabelRequired(fieldLabel = "Color");
+
+		FormPortlet.waitFormRenderer(text = "Submit");
+
+		FormPortlet.submitWithValidationError(validationErrorMessage = "This field is required.");
+
+		FormViewBuilder.editColor(
+			fieldName = "Color",
+			fieldValue = "FF0D0D");
+
```
Comment (carolmariaabb) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsColorField.testcase]: @leticiachiu Can you please use the `FormFields.editColor` macro instead? Thanks!

**Comment by leticiachiu** [comment]
fixed and rebased.

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
@leticiachiu SF failure

**Comment by leticiachiu** [comment]
fixed

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**